src/test/java/uk/gov/ons/ctp/response/sample/endpoint/CsvIngesterBusinessTest.java

```diff
@@ -48,93 +52,141 @@
-  private static final String TRADSTYLE2 = "TRADSTYLE2_COMPANY1";
-  private static final String TRADSTYLE3 = "TRADSTYLE3_COMPANY1";
-  private static final String SELTYPE = "C";
-  private static final String INCLEXCL = "D";
-  private static final String CELLNO = "7";
-  private static final String FORMTYPE = "15";
-  private static final String CURRENCY = "S";
-
-  /**
-   * take a named test file and create a copy of it - is because the ingester
-   * will delete the source csv file after ingest
-   *
-   * @param fileName source file name
-   * @return the newly created file
-   * @throws Exception oops
-   */
-  private MockMultipartFile getTestFile(String fileName) throws Exception {
-    Path csvFileLocation = Paths.get(getClass().getClassLoader().getResource("csv/" + fileName).toURI());
-    MockMultipartFile multipartFile = new MockMultipartFile("file", fileName, "csv",
-        Files.readAllBytes(csvFileLocation));
-    return multipartFile;
-  }
 
   @Test
-  public void testBlueSky() throws Exception {
+  public void testIngestBusinessSampleSuccess() throws Exception {
+    // Given
     SampleSummary sampleSummary = new SampleSummary();
-    csvIngester.ingest(sampleSummary, getTestFile("business-survey-sample.csv"));
-    verify(sampleService, times(1)).processSampleSummary(eq(sampleSummary),
-        anyListOf(BusinessSampleUnit.class));
+    sampleSummary.setId(UUID.randomUUID());
+    BusinessSampleUnit businessSampleUnit = createBusinessSampleUnit();
+    given(sampleService.saveSample(sampleSummary, Collections.singletonList(businessSampleUnit), SampleUnitDTO.SampleUnitState.INIT)).willReturn(sampleSummary);
+
+    // When
+    csvIngester.ingest(sampleSummary, TestFiles.getTestFile("business-survey-sample.csv"));
+
+    // Then
+    verify(sampleService, times(1)).saveSample(eq(sampleSummary),
+        anyListOf(BusinessSampleUnit.class), eq(SampleUnitDTO.SampleUnitState.INIT));
   }
 
   @Test
-  public void testMultipleLines() throws Exception {
+  public void testParseMultipleLinesFromBusinessSampleFile() throws Exception {
+    // Given
     SampleSummary sampleSummary = new SampleSummary();
-    csvIngester.ingest(sampleSummary, getTestFile("business-survey-sample-multiple.csv"));
-    verify(sampleService, times(1)).processSampleSummary(eq(sampleSummary),
-            anyListOf(BusinessSampleUnit.class));
+    sampleSummary.setId(UUID.randomUUID());
+    given(sampleService.saveSample(any(), any(), eq(SampleUnitDTO.SampleUnitState.INIT))).willReturn(sampleSummary);
+
+    // When
+    csvIngester.ingest(sampleSummary, TestFiles.getTestFile("business-survey-sample-multiple.csv"));
+
+    // Then
+    verify(sampleService, times(1)).saveSample(eq(sampleSummary),
+            anyListOf(BusinessSampleUnit.class), eq(SampleUnitDTO.SampleUnitState.INIT));
   }
 
   @Test(expected = CTPException.class)
-  public void testDate() throws Exception {
+  public void testParseDateFromSampleSummary() throws Exception {
```

Comment:
I think this is trying to test parsing invalid date, might be worth adding it to the name